protected/tests/support/BrowserSignInSteps.php

```diff
@@ -21,4 +21,5 @@ public function loginToWebSiteWithSessionAndCredentialsThenAssert($email,$passwo
 
         $this->assertTrue($this->session->getPage()->hasContent($to_assert));
 	}
-}
\ No newline at end of file
+
```

Comment:
Looks like the changes to this file are a couple of newlines. Maybe we don't need to merge these changes?

Reply:
Yes, please don't merge them.